src/moveFile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const fs = require('fs').promises;
+const path = require('path');
+
+async function moveFile(fileName, moveTo) {
+  const filePath = path.join(__dirname, fileName);
+  let movePath = path.join(__dirname, '..', moveTo);
+  let data;
+
+  movePath = movePath.endsWith('/') ? movePath.slice(0, -1) : movePath;
+
+  fs.readFile(movePath, 'utf-8', (err) => {
```

Comment:
fs.readFile() is used with a callback, but the code is in an async function. You should use await with fs.readFile() or manage the promise returned by fs.readFile().